python/py_vapid/__init__.py

```diff
@@ -29,7 +31,10 @@ def __init__(self, private_key_file=None, private_key=None):
         private key
         """
         if private_key_file:
-            private_key = open(private_key_file).read()
+            if not os.path.isfile(private_key_file):
+                self.save_key(private_key_file)
+                return
+            private_key = open(private_key_file, 'r').read()
```

Comment:
@jrconlin If these changes could be pulled in too that would be great. It allows users to specify the private_key_file when they first load py-vapid and it will be created if a file doesn't exist at the path.